Tests/test_tuple.py

```diff
@@ -47,13 +47,13 @@ def test_add_mul(self):
         self.assertEqual((1,2,3) * 2, (1,2,3,1,2,3))
         self.assertEqual(2 * (1,2,3), (1,2,3,1,2,3))
 
-        class mylong(long): pass
+        class mylong(int): pass
```

Comment:
I thought about it too but tried to restraint myself from too much cleanup in this PR to keep is as small as possible, but also because the idea of cleaning up tests can become a big distraction, there is so much that can be cleaned up and it is tempting.

In this particular case, if you just object to the name `mylong`, I have planned a pass on that in my follow ups. Since Python `long` is gone, `long` can only be read as `Int64` and better not used at all except as a C# type.

If you meant using `myint` from `type_util`, then I was not planning of using it here, because the test also defines/uses `mylong2` and I think it is more explicit about the test intentions to have both types defined side-by-side. So here probably I would use names `myint1`/`myint2` instead but keep the class definitions.